server/src/daemon/item.rs

```diff
@@ -0,0 +1,27 @@
+// org.freedesktop.Secret.Item
+
+use oo7::dbus::api::SecretInner;
+use zbus::zvariant::{ObjectPath, OwnedObjectPath};
+
+use super::Result;
+
+#[derive(Debug)]
+pub struct Item {
+    _path: OwnedObjectPath,
+}
+
+#[zbus::interface(name = "org.freedesktop.Secret.Item")]
+impl Item {
+    pub async fn delete(&self) -> Result<ObjectPath> {
+        todo!()
+    }
+
+    #[zbus(name = "GetSecret")]
```

Comment:
The methods should name the in & out arguments so they are correctly displayed on apps like d-spy

Reply:
I couldn't find a way to name `IN` arguments. but, I have named all the `OUT` args.

Reply:
The in argument are the name you give to the function parameters

Reply:
got it, thanks!